tests/unit/linalg/test_hypre_ilu.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "unit_tests.hpp"
+#include "mfem.hpp"
+
+namespace mfem
+{
+
+#ifdef MFEM_USE_MPI
```

Comment:
Should also include hypre version guard here, because this test does not compile with hypre < 2.19.

Reply:
Good catch, I'm already using 2.19 on my builds so I didn't notice.